lib/core/api/fetch-wrapper.ts

```diff
@@ -0,0 +1,79 @@
+import { mapFetchErrorToHttpException } from "./fetch-exception.mapper";
+import { ServerErrorPayload } from "@lib/core/error/http/http-exception";
+import { HttpExceptionFactory } from "@lib/core/error/http/http-exception.factory";
+
+async function http<T>(path: string, config: RequestInit): Promise<T> {
+  const request = new Request(path, config);
+  const response: Response = await fetch(request);
```

Comment:
I think if errors are aborted this actually throws, but react-query handles that so maybe we don't have to care about it